sdk/eventhub/azure-eventhub/stress/templates/testjob.yml

```diff
@@ -0,0 +1,16 @@
+{{- include "stress-test-addons.deploy-job-template.from-pod" (list . "stress.python-eh-stress") -}}
+{{- define "stress.python-eh-stress" -}}
+metadata:
+  labels:
+    testName: "deploy-python-eh-stress"
+    testInstance: "deploy-python-eh-stress-instance"
+    chaos: "true"
+spec:
+  containers:
+    - name: python-eh-stress
+      image: stresstestregistry.azurecr.io/yuling/py-eh-stress:v2
```

Comment:
The labels/image should be updated to be guaranteed unique for our auto-release pipelines (no `testInstance` targeting overlaps with chaos, and a dynamic image name that matches what gets built by the scripts in the pipeline).

```
  labels:
    testName: "deploy-python-eh-stress"
    testInstance: "eventhub-{{ .Release.Name }}-{{ .Release.Revision }}"
    chaos: "true"
spec:
  containers:
    - name: python-eh-stress
      image: {{ default "stresstestregistry.azurecr.io/python" .Values.repository }}/stress:{{ default "v2" .Values.tag }}
```

Reply:
The `testInstance field can be whatever you want, but it has to include both `.Release.Name` and `.Release.Revision`.